include/deal.II/base/mpi_consensus_algorithm.h

```diff
@@ -28,553 +28,556 @@ namespace Utilities
 {
   namespace MPI
   {
-    /**
-     * An interface to be able to use the ConsensusAlgorithm classes. The main
-     * functionality of the implementations is to return a list of process ranks
-     * this process wants data from and to deal with the optional payload of the
-     * messages sent/received by the ConsensusAlgorithm classes.
-     *
-     * There are two kinds of messages:
-     * - send/request message: A message consisting of a data request
-     *   which should be answered by another process. This message is
-     *   considered as a request message by the receiving rank.
-     * - recv message: The answer to a send/request message.
-     *
-     * @tparam T1 the type of the elements of the vector to sent
-     * @tparam T2 the type of the elements of the vector to received
-     *
-     * @note Since the payloads of the messages are optional, users have
-     *       to deal with buffers themselves. The ConsensusAlgorithm classes 1)
-     *       deliver only references to empty vectors (of size 0)
-     *       the data to be sent can be inserted to or read from, and
-     *       2) communicate these vectors blindly.
-     *
-     * @author Peter Munch, 2019
-     */
-    template <typename T1, typename T2>
-    class ConsensusAlgorithmProcess
+    namespace ConsensusAlgorithm
```

Comment:
I had actually thought of using `ConsensusAlgorithms` as the namespace name (plural). That would match `DoFTools`, `FETools`, etc.

Reply:
I prefer it in singular because you can read for example `ConsensusAlgorithm::Process` simply as consensus-algorithm process.


What do others think?

Reply:
have No strong opinion, but I would have picked plural as well.

Reply:
I guess I am overruled. I will make the change after PR #9630 is merged.

Reply:
I have made the changes.